src/components/Trigger/Trigger.tsx

```diff
@@ -64,11 +62,11 @@ export const Trigger: FC<TriggerProps> = ({
                         data-test-id="dropdown-clear-button"
                         aria-label="Clear selection"
                         className={merge([
-                            "tw-p-0 tw-outline-none",
+                            "tw-p-0 tw-outline-none tw-absolute tw-right-5",
```

Comment:
is this necessary? the parent container is a flexbox which provides the spacing between both buttons. I would assume that maybe the tw-absolute can be removed from the clear button and still work as intended

Reply:
Without these styles, the small gap between the clear icon and the caret icon will not be clickable as shown in the "before" video of the PR.
If that's okay with you guys then it can be removed.

Reply:
Ah you are correct 👍